service/controller/v14/cluster_resource_set.go

```diff
@@ -371,24 +370,42 @@ func NewClusterResourceSet(config ClusterResourceSetConfig) (*controller.Resourc
 		}
 	}
 
-	var statusResource controller.Resource
-	{
-		c := statusresource.Config{
-			ClusterStatusFunc:        key.ToClusterStatus,
-			NodeCountFunc:            key.ToNodeCount,
-			Logger:                   config.Logger,
-			RESTClient:               config.G8sClient.ProviderV1alpha1().RESTClient(),
-			VersionBundleVersionFunc: key.ToVersionBundleVersion,
-		}
-
-		statusResource, err = statusresource.New(c)
-		if err != nil {
-			return nil, microerror.Mask(err)
-		}
-	}
+	//var guestCluster guestcluster.Interface
+	//{
+	//	c := guestcluster.Config{
+	//		CertsSearcher: certsSearcher,
+	//		Logger:        config.Logger,
+	//
+	//		CertID: certs.APICert,
+	//	}
+	//
+	//	guestCluster, err = guestcluster.New(c)
+	//	if err != nil {
+	//		return nil, microerror.Mask(err)
+	//	}
+	//}
+
+	//var statusResource controller.Resource
+	//{
+	//	c := statusresource.Config{
+	//		ClusterEndpointFunc:      key.ToClusterEndpoint,
+	//		ClusterIDFunc:            key.ToClusterID,
+	//		ClusterStatusFunc:        key.ToClusterStatus,
+	//		GuestCluster:             guestCluster,
+	//		NodeCountFunc:            key.ToNodeCount,
+	//		Logger:                   config.Logger,
+	//		RESTClient:               config.G8sClient.ProviderV1alpha1().RESTClient(),
+	//		VersionBundleVersionFunc: key.ToVersionBundleVersion,
+	//	}
+	//
+	//	statusResource, err = statusresource.New(c)
+	//	if err != nil {
+	//		return nil, microerror.Mask(err)
+	//	}
+	//}
 
 	resources := []controller.Resource{
-		statusResource,
+		//statusResource,
```

Comment:
I'm playing catch up but I miss some context on why the statusResource is being disabled. I think a comment would be useful.

Reply:
+1 interesting :) Also happened after my review.

Reply:
I did this like ten minutes ago. I will provide information when I am ready.

Reply:
Please have another look. I added a TODO comment and an issue for it. See https://github.com/giantswarm/giantswarm/issues/3822.

Reply:
I don't see the comment. Can you check?

Reply:
Oh yeah, when I push the code you see the comment. :D Please check now.